media/static-bundles.json

```diff
@@ -1636,6 +1636,8 @@
       "files": [
         "js/libs/jquery-3.2.1.min.js",
         "js/base/mozilla-utils.js",
+        "protocol/js/protocol-newsletter.js",
```

Comment:
Including these files in the base bundle here causes the scripts to throw errors on pages where the newsletter isn't included:

```
TypeError: Argument 1 of Window.getComputedStyle is not an object.
protocol-newsletter.js:16
TypeError: Mozilla.Client is undefined
```

Rather than increase the bundle size for all pages on the site, would it be more performant to create a common newsletter bundle so that it can be included only on the pages where it's needed?

Reply:
I'm not sure how to know if it's needed.

Is it okay if I include the `<script>` in the `{% block newsletter %}` block? It'd be a `<script>` tag part way up the page but removed when this block is blank, and it could be async to avoid being render blocking.

Totally separate from how to include it... the script should not throw an error in this case and I'll fix that.

Reply:
Alternatively, I think i need to make a `protocol-basic.js`, I can put it in there.

Reply:
A protocol-basic bundle would be my preferred way to go.